okhttp/src/main/kotlin/okhttp3/ConnectionPool.kt

```diff
@@ -92,4 +125,34 @@ class ConnectionPool internal constructor(
   fun evictAll() {
     delegate.evictAll()
   }
+
+  /**
+   * Sets a policy that applies to [address].
+   * Overwrites any existing policy for that address.
+   */
+  @ExperimentalOkHttpApi
+  fun setPolicy(
+    address: Address,
+    policy: AddressPolicy,
+  ) {
+    delegate.setPolicy(address, policy)
+  }
+
+  /**
+   * A policy for how the pool should treat a specific address.
+   */
+  class AddressPolicy(
+    /**
+     * How many concurrent calls should be possible to make at any time.
+     * The pool will routinely try to pre-emptively open connections to satisfy this minimum.
+     *
+     * This feature must not be used on Android because it may result in network traffic while
+     * the app is backgrounded. On Android, leave this set to 0.
```

Comment:
I don’t know about this comment. It might be true, but Android is not necessarily always deployed in the exact same way. It’s possible for an embedded Android to have 24-7 electricity and this would be a reasonable behavior there.

Reply:
I'll move the code comment and we can choose to address it (or not) in the public docs